src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java

```diff
@@ -516,7 +516,7 @@ public List<OffenderCategorise> getCategory(final String agencyId, final Categor
         };
     }
 
-    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "MAINTAIN_ASSESSMENTS"})
```

Comment:
These endpoints do not have SYSTEM_USER in the resource file PreAuthorize method so is already tested and can therefore be removed